decide/administration/frontend/src/api/authApiUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { axios } from "api/axios";
+
+const authApi = {
+  login: (username: string, password: string) => {
+    return axios.post("/auth/login", {
```

Comment:
Se debería usar la integración con la api de autenticación /authentication/login_form para logear a los usuarios

Reply:
El frontend de administración mantiene el estado _"logged in"_ basandose en una cookie que se setea en el modulo de administration y por tanto se requiere el uso de dicho endpoint para el log in.

Deberiamos realizar la integración a nivel de backend.